python/sglang/srt/distributed/communication_op.py

```diff
@@ -0,0 +1,34 @@
+# Adapted from https://github.com/vllm-project/vllm/blob/a6221a144af772fd1a68fe7e627935dc53e81738/vllm/distributed/communication_op.py
+from typing import Any, Dict, Optional, Union
+
+import torch
+import torch.distributed
+
+from .parallel_state import get_tp_group
```

Comment:
use absolute path.

Reply:
OK，I will fix it when I add custom_ar to sgl-kernel